setup-env

```diff
@@ -191,9 +176,6 @@ else
     echo "Your configuration files at $BUILDDIR have not been touched."
 fi
 
-# sets TEMPLATECONF
-. $TDROOT/.templateconf
```

Comment:
Did this get dropped for a reason?  Shouldn't it have been moved to just under where `TEMPLATECONF` get set (line 164)?

Reply:
This same functionality is moved to line 164 (`TEMPLATECONF=$(cat "$BUILDDIR/conf/templateconf.cfg")` it is set from the $BUILDDIR/conf/templateconf.cfg which doesn't require the `DISTRO` variable. It had to be moved up as it's required to cat the conf-notes.txt.

Reply:
Right, but if `templateconf.cfg` doesn't exist yet (i.e., new build dir) sourcing the `.templateconf` file will set it appropriately. Having both mimics the sequence in `oe-setup-builddir`, which I think is what we want here.

Reply:
`oe-setup-builddir` is run prior to this step, and it is responsible for writing `templateconf.cfg` with the interpolated value it generated from `.templateconf` before we try to use it here.

Reply:
Ah, OK. That's the bit I was missing just looking at the diffs.